CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v1.4.0 (05 May 2017)
+
+This release adds the ability to customize the line endings used in zap's log
+output. It is fully backward-compatible.
```

Comment:
nit - Probably dont need to mention it's backwards-compatible, that's implied by the fact that 'dis is a minor.